src/PanZoom.js

```diff
@@ -670,15 +709,54 @@ class PanZoom extends React.Component<Props> {
     }
   }
 
+  _setContainerRef = (ref: HTMLDivElement | null) => { 
```

Comment:
@n1ru4l What do you think about using `React.createRef()` instead of function to set the refs?